source/index.html.md

```diff
@@ -1,185 +1,1260 @@
+
+Name | Format | Description
+-------- | --------- | ---------
+name | String | The name of the tag.
+**Optional elements** |
+enabled | Boolean | Whether the tag is enabled or not. Default value is `true`.<br>*Note:* When multi-level tagging is used, this value is ignored and is consider `true`.
+glCode | String | The GL Code associated to the tag.
+
+
+#### Passing tag data from plain text as additional request parameters (dependent or independent tag levels)
+
+The tag data must be passed in a parameter called `file` in the request.
+
+The `tags` object must contain the following information:
+
+Name | Format | Valid values
+-------- | --------- | ---------
+source | String | "file" |
+config | JSON Object | See below |
+
+
+- `tags.config`
+
+Name | Format | Valid values | Description
+-------- | --------- | ---------------- | ---------
+dependency | Boolean | true, false | Whether tag level are dependent. |
+glCodes | Boolean | true, false | Whether adjacent columns in the tag file(s) contain GL Codes.
+header | Boolean | true, false | Whether the first line of the file contains the names of the tag levels.
+setRequired | Boolean | true, false | If set to `true`, users will be required to tag each expenses under that category.
+fileType | String | "cvs" or "tsv" | Format of the tag data.
+
+## Employee updater
+
+```shell
+curl -X POST 'https://integrations.expensify.com/Integration-Server/ExpensifyIntegrations' \
+    -d 'requestJobDescription={
+        "type": "update",
+        "credentials": {
+            "partnerUserID": "_REPLACE_",
+            "partnerUserSecret": "_REPLACE_"
+        },
+        "inputSettings": {
+            "type": "employees",
+            "policyID":"0123456789ABCDEF",
+            "fileType": "csv"
+        }
+    }'
+    --data-urlencode 'data@employeeData.csv'
+```
+
+> Employee data samples - employeeData.csv
+
+> - Basic information
+
+```
+EmployeeEmail,ManagerEmail,Admin
+user1@domain.com,manager1@domain.com,false
+user2@domain.com,manager1@domain.com,true
+manager2@domain.com,manager1@domain.com,true
+```
+
+> - All supported fields
+
+```
+EmployeeEmail,ManagerEmail,Admin,ForwardManagerEmail,EmployeeUserId,EmployeePayrollId
+user1@domain.com,manager1@domain.com,false,cfo@domain.com,User1ID,User1PayrollID
+user2@domain.com,manager1@domain.com,true,cfo@domain.com,,User2PayrollID
+manager2@domain.com,manager1@domain.com,true,,Manager1ID,
+```
+
+> Response
+
+> - A success response message is comprised of a `responseCode` 200, and the total number of employees after the update.
+
+```
+{
+    "responseCode": 200,
+    "nbEmployees": 42
+}
+```
+
+> - Error
+
+```
+{
+    "responseMessage": "Column EmployeeEmail not found in CSV header",
+    "responseCode": 500
+}
+```
+
+Lets you manage employees on an Expensify policy. Use this to upload employee data in order to configure or provision accounts for new hires.
+
+The employee data must be passed in the request parameter `data`.
+
+- `inputSettings`
```

Comment:
Use this to configure or provision accounts for new hires, as well as to update existing employee data